LeaderboardBackend/Controllers/RunsController.cs

```diff
@@ -107,4 +110,25 @@ public async Task<ActionResult<List<Participation>>> GetParticipations(Guid id)
 
 		return Ok(participations);
 	}
+
+	[ApiConventionMethod(typeof(Conventions), nameof(Conventions.Get))]
+	[HttpGet("{id}/category")]
+	public async Task<ActionResult<Category>> GetCategoryForRun(Guid id)
+	{
+		Run? run = await _runService.GetRun(id);
+
+		if (run is null)
+		{
+			return NotFound("Run not found");
+		}
+
+		Category? category = await _categoryService.GetCategoryForRun(run);
+
+		if (category is null)
+		{
+			return NotFound("Category not found");
+		}
+
+		return Ok(category);
```

Comment:
Like I guess I don't mind having this implementation, though I am considering having `_runService.GetRun()` be able to return its relationships. That way, you could do something like a (skipping error handling): 
```cs
Run? run = await _runService.GetRun(id, true); // second arg to get relationships
return run.Category;
```
which'll save us the second (honestly tiny) DB call.
@RageCage64 thoughts?

Reply:
Could do it and does make sense this way. Just did it like this, because GetRun didn't return the relationships and I didn't think of your solution